pallets/parachain-staking/src/mock.rs

```diff
@@ -67,7 +67,6 @@ construct_runtime!(
 
 parameter_types! {
 	pub const BlockHashCount: u64 = 250;
-	pub const MaximumBlockWeight: Weight = 1024;
```

Comment:
Yeah. I removed that line to allow tests to run. 
`cargo test` fails on `main` because weight now expects a struct.

`MaximumBlockWeight ` has been removed from KILT's mock file:
https://github.com/KILTprotocol/kilt-node/blob/develop/pallets/parachain-staking/src/mock.rs#L67